### DISCUSSION TITLE
How to setup Airflow Dev setup in windows 10 ?

### DISCUSSION THREAD

**Discussion by ramjeeanna** [apache/airflow#14234] [Q&A] [answered]
Hi,
Am new to the airflow entirely.  I have setup the airflow code in Linux machine and able to run it.
In windows, how to set up the code in pytcharm and run from IDE ? 

Any detailed steps would be a great help to me

**Comment by potiuk** [comment, marked as answer]
You have to use WSL2 to run airflow, but in Windows you need to simply install airflow in you local virtualenv (you can use it in your IDE but you won't be able to run it).

There are some instructions contributed by Windows contributors here on how to setup WSL2 for execution: https://github.com/apache/airflow/blob/master/BREEZE.rst#docker-in-wsl-2

Full support for either running or executing Aiflow on Windows is something that several people would like to have, there are open issues for it #10388 and here #12874 - but no-one works actively on it at this moment. But if someone who uses Windows would like to do it - then any help is welcome.
